index.html

```diff
@@ -0,0 +1,235 @@
+
+<body>
+
+    <!-- Nav bar Section -->
+
+    <div class="nav-bar">
+                <div class="contact-button">
+                    <button>+00993390090 </button>
+                </div>
+      <div class="text">
+           <h1>welcome!</h1>
+           <p>let us help build your dream project</p>
+
+      </div>
+        
+    </div>
+
+
+
+    <!--  Section Two -->
+
+
+
+    <section class="Section-two-section">
+        <div class="section-two">
+            <div class="header">
+                <h1>How can we help you?</h1>
+                <p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Suspendisse et<br> justo. Praesent mattis commodo augue.</p>
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-object-ungroup"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-code"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-envelope-open-text"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+            </div>
+
+
+            <div class="container">
+
+                <div class="box">
+
+                    <h2><i class="fas fa-lightbulb"></i> <san></san>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-cash-register"></i> <span></span>Design</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+                <div class="box">
+
+                    <h2><i class="fas fa-user-tie"></i> spanDesign</h2>
+                    <pr>Lorem ipsum dolor sit amet, <br>
+                        consectetuer adipiscing elit.<br> Suspendisse et justo. Praesent mattis<br> commodo augue</pr>
+
+                </div>
+
+            </div>
+
+        </div>
+    </section>
+
+
+    <section class="section-three-section">
+
+        <div class="section-three">
+            <div class="section-three-head">
+                <h1>Portfolio</h1>
```

Comment:
Having more than one h1 tags is not in line wth the best practice. kindly rectify